File: Amerike-2dproyect-25/Assets/BoundleAssets/CharacterA/scripts/character/Models/characterData.cs
using UnityEngine;
using System;
namespace Character.Models
{
    [Serializable]
    public class characterData : ICharacterData
    {
        [SerializeField]
        private string styleName;
        [SerializeField]
        private float moveSpeed;
        [SerializeField]
        private float jumpforce;
        public float MoveSpeed { get; }
        public float JumpForce { get;  }

        public String StyleName { get; }
        
        //public CharacterData(float moveSpeed, float jumpForce)
        //public float JumpForce { get => throw new System.NotImplementedException(); set => throw new System.NotImplementedException(); }
    }
}

File: Amerike-2dproyect-25/Assets/WebPart/ClaseMotoresAvanzados/src/models/characterData.js
import mongoose from "mongoose";

const schema = new mongoose.Schema({
  MoveSpeed:{
    type: Number,
    required: true,
    unique: false
  },
  JumpForce:{
    type: Number,
    required: true,
    unique: false
  },
  StyleName:{
    type: String,
    required: true,
    unique: false
  }
});

export default mongoose.model("CharacterData",schema);